packages/@secretlint/formatter/src/index.ts

```diff
@@ -104,5 +153,22 @@ export interface SecretLintFormatterDetail {
 }
 
 export function getFormatterList(): SecretLintFormatterDetail[] {
-    return textlintGetFormatterList();
+    const textlintFormatters = textlintGetFormatterList();
+    const secretlintFormatters = secretlintGetFormatterList();
+    const secretlintFormatterNames = secretlintFormatters.map((formatter) => formatter.name);
+    const externalFormatters = textlintFormatters.filter(
+        (formatter) => !secretlintFormatterNames.includes(formatter.name)
+    );
+    return [...externalFormatters, ...secretlintFormatters];
+}
+
+export function secretlintGetFormatterList(): SecretLintFormatterDetail[] {
+    return fs
+        .readdirSync(path.join(__dirname, "formatters"))
+        .filter((file: string) => {
+            return path.extname(file) === ".ts";
```

Comment:
Probably, we also need to treat `.js` ext.
Testing code work as `.ts`, but distribution code work as `.js`.